nautobot/dcim/models/racks.py

```diff
@@ -52,7 +53,7 @@ class RackGroup(MPTTModel, OrganizationalModel):
     """
 
     name = models.CharField(max_length=100)
-    slug = models.SlugField(max_length=100)
+    slug = AutoSlugField(populate_from="name", max_length=100, editable=True, overwrite_on_add=False)
```

Comment:
Same :sad: here about not being globally unique, but correct to not change the behavior at this time.

Reply:
unique_together meta (site+name and site+slug)